src/styles/blocks/star.scss

```
@@ -0,0 +1,42 @@
+
+.card {
+  display: flex;
+  align-items: center;
+  padding-top: 16px;
+  justify-content: space-between;
+
+  &__review {
+    color: var(--Main-accent);
+    text-align: right;
+    font-size: 10px;
+    font-weight: 400;
+    padding: 0;
+    margin: 0;
+  }
+}
+
+.stars {
+  display: flex;
+  align-items: center;
+
+  &__star {
+    background-image: url("../../../../../../projects/layout_product-cards/src/images/star.svg");
```

```suggestion
    background-image: url("../images/star.svg");
```

Try to use this path instead